### ISSUE TITLE
AIX Support?

### ISSUE THREAD

**Issue by kurtkilpela** [GemTalk/FileSystemGs#104]
Martin requested we add support for FileSystem on AIX. Previously, we determined that we wouldn't support FileSystem on AIX and Solaris.

**Comment by kurtkilpela** [comment]
Per internal meeting, AIX is a nice to have. Historically, there have been issues w/ FFI on AIX. Once everything else is done, I'll evaluate supporting AIX.
